lib/view_model/after_auth_view_models/profile_view_models/edit_profile_view_model.dart

```diff
@@ -63,25 +61,6 @@ class EditProfilePageViewModel extends BaseModel {
     }
   }
 
-  /// This function is used to convert the image into Base64 format.
-  ///
-  /// **params**:
-  /// * `file`:  Takes the image in format of file.
-  ///
-  /// **returns**:
-  /// * `Future<String>`: image in string format
-  Future<String> convertToBase64(File file) async {
-    try {
-      final List<int> bytes = await file.readAsBytes();
-      final String base64String = base64Encode(bytes);
-      print(base64String);
-      imageFile = base64String as File?;
-      return base64String;
-    } catch (error) {
-      return '';
-    }
-  }
-
```

Comment:
Why we need to remove this function ?

Reply:
the above function is not being used in edit_profile section, but is being used in add_profile section, and so i have removed it from here and added corresponding tests in add_profile section

Reply:
@vaidic-dodwani where is this code moved ? No file change shows the addition. Moreover, converting files to base64 seems to be a common feature that can be used by many files, so it'll be better to have this func in a separate service (say ImageService), instead of moving to specific files.

Reply:
@vaidic-dodwani where is this code moved ? No file change shows the addition. Moreover, converting files to base64 seems to be a common feature that can be used by many files, so it'll be better to have this func in a separate service (say ImageService), instead of moving to specific files.

Reply:
@literalEval The code segment isn't moved, it was already present where it is required, it didn't have tests which are now added.
Regarding the separate service, I think the function isn't used anywhere else so it doesn't make a great difference, in future if it is required I think then we should make a ImageService.